scrabbler/strategy.py

```diff
@@ -4,8 +4,12 @@ class Strategy:
     Class representing a player's Strategy. The three algorithms should be a 
     subclass of this class and override the method choose move.
     '''
+    def choose_move(self, game, rack):
```

Comment:
Why do you have this here?

Reply:
Python does not support proper abstract classes and methods, so this is the next best thing. `choose_move` here is meant to an abstract method, so this enforces that any class that extends `Strategy` must implement this method. This is in line with the way "abstract" methods and classes are defined in the Pacman projects.